src/diagram.ts

```diff
@@ -25,13 +26,19 @@ export function registerDiagramCommands(context: vscode.ExtensionContext) {
 		if(activeEditor) {
 			const cfg = await (await getFlowrSession()).retrieveCfgMermaid(activeEditor.document)
 			if(cfg) {
-				createWebview('flowr-cfg', 'Control Flow Graph', cfg)
+				createWebview('flowr-cfg', 'Control Flow Graph', cfg, output)
 			}
 		}
 	}))
 }
 
-function createWebview(id: string, name: string, mermaid: string) : vscode.WebviewPanel {
+function createWebview(id: string, name: string, mermaid: string, output: vscode.OutputChannel) : vscode.WebviewPanel | undefined {
+	if(mermaid.length > mermaidMaxTextLength()){
+		void vscode.window.showErrorMessage('The diagram is too large to be displayed by Mermaid. You can find its code in the flowR output panel instead.')
```

Comment:
1) is this the same way mermaid measures text length?
2) We should add a note that the size is configurable.

Reply:
1. yes :)
2. ok!